### ISSUE TITLE
general remark -- log levels

### ISSUE THREAD

**Issue by j3k0** [j3k0/ganomede#11]
@elmigranto 

Here is small issue I have when reading logs to figure out server issues. Till now, we used `log.error` for whatever error happens, from "login failed wrong credentials", "invalid move", "database connection failures" to "uncaught exceptions". Filtering the important errors from all of the noise is pretty hard.

So from now on, I try to be more cautious on my usage of log levels. Errors like "user sent invalid credentials" are not really system errors, so they can be logged with `log.info`. Uncaught exceptions are `log.fatal` (because worker has to restart). etc.

Bunyan allows more control over levels, and suggest some best practice that I'd like that we follow.

Maybe the boilerplate's Error might also provide an optional `severity` field, which default to `"error"` but can be changed depending on each error.

## snip-snap (the sound of copy-paste)
The log levels in bunyan are as follows. The level descriptions are best practice opinions of the author.

 * "fatal" (60): The service/app is going to stop or become unusable now. An operator should definitely look into this soon.
 * "error" (50): Fatal for a particular request, but the service/app continues servicing other requests. An operator should look at this soon(ish).
 * "warn" (40): A note on something that should probably be looked at by an operator eventually.
 * "info" (30): Detail on regular operation.
 * "debug" (20): Anything else, i.e. too verbose to be included in "info" level.
 * "trace" (10): Logging from external libraries used by your app or very detailed application logging.

**Comment by elmigranto** [comment]
Yeah, I like this, good idea. Solves "to log or not to log" inside [`sendHttpError`](https://github.com/j3k0/ganomede-boilerplate/blob/boilerplate.js/src/errors.js#L96) too!

More error-related stuff:
  - https://github.com/j3k0/ganomede-boilerplate/issues/10 — important
  - https://github.com/j3k0/ganomede-boilerplate/issues/5 — w/ever
